### PR TITLE
Update fr.json

### PR DESCRIPTION
**Pull request by itasli**
corrected some typos and translation

### CODE REVIEW

**Comment by maxigator** [comment]
There is a new field at line 35:
`"stillcurious": "Still curious? <a href='about.html'>Learn More</a><link rel='prefetch' href='/about'>."`

**Comment by itasli** [comment]
added the new line 👍

File: locales/fr.json [hunk 1]
```diff
@@ -32,8 +32,9 @@
...
           "screenshottitle2": "Rendu d'entité efficace",
           "screenshotdesc2": "Iris améliore la façon dont Minecraft affiche les entités et blocks les entités comme les coffres, les cochons et les cadres d'objets. Cette astuce fonctionne même avec les shaders désactivés, et dans les mondes avec beaucoup d'entités, elle peut augmenter les performances de Vanilla jusqu'à 60%. Le rendu des ombres connaît une amélioration particulièrement importante, transformant des zones autrefois injouables avec les shaders en des endroits où votre FPS est étonnamment fluide !<br><span class='text-span-2'>Capture d'écran par kinzuu_music</span>",
-          "footer1": "À propos du projet Iris",
-          "footerdesc1": "Le projet Iris est un projet collaboratif open source créé par une équipe de développeurs passionnés cherchant à faire en sorte que l'éxperience des shaders Minecraft est la meilleure possible. Iris est développé par la communauté, pour la communauté.",
+          "stillcurious": "Curieux ?<a href='about.html'>En savoir plus</a><link rel='prefetch' href='/about'>.",
+           "footer1": "À propos du projet Iris",
```
Comment (maxigator) [inline locales/fr.json:36]:
```suggestion
          "stillcurious": "Curieux ? <a href='about.html'>En savoir plus</a><link rel='prefetch' href='/about'>.",
          "footer1": "À propos du projet Iris",
```

**Review by maxigator — CHANGES_REQUESTED** [review]
remove space and add one too

**Review by maxigator — APPROVED** [review]

**Comment by thedocruby** [comment]
Thanks guys!

**Pull request merged**